### PR TITLE
Added some style preferences to prettier and eslint

### PR DESCRIPTION
**Pull request by NoahSaso**
Triple equals for the win!

### CODE REVIEW

File: packages/config/eslint/index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const eslintConfig = {
     'no-unused-vars': ['off'],
     'react/jsx-sort-props': ['warn', { reservedFirst: ['key'] }],
     'tailwindcss/classnames-order': ['warn'],
+    eqeqeq: ['error'],
```
Comment (ebaker) [inline packages/config/eslint/index.js:21]: nice :+1:

File: packages/config/prettier/index.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ const prettierConfig = {
   tabWidth: 2,
   trailingComma: 'es5',
   useTabs: false,
+  bracketSpacing: true,
```
Comment (ebaker) [inline packages/config/prettier/index.js:10]: cool :+1:

File: packages/config/prettier/index.js [hunk 3]
```diff
@@ -7,6 +7,9 @@ const prettierConfig = {
...
+  arrowParens: 'always',
```
Comment (ebaker) [inline packages/config/prettier/index.js:11]: not my fav, but i think it make sense for typescript :+1:

**Review by ebaker — APPROVED** [review]
lgtm :+1:

**Review by JakeHartnell — APPROVED** [review]

**Comment by baoskee** [comment]
Still a double equal rebel. Nice try Noah

**Pull request merged**